Tests/XcodeGenKitTests/ProjectGeneratorTests.swift

```diff
@@ -274,10 +274,15 @@ class ProjectGeneratorTests: XCTestCase {
                 //     - framework: FrameworkA.framework
                 //     - framework: FrameworkB.framework
                 //       embed: false
+                // iOSFrameworkZ:
```

Comment:
This test can probably me moved into a seperate `testDependencies` test function later